api-docs/v2/getting-started/install-CLI-client.rst

```diff
@@ -23,7 +23,7 @@ underneath your current directory.
       prod:
         auth:
           auth_url: https://identity.api.rackspacecloud.com/v2.0/
-          project_id: 123456
+          project_id: <RACKSPACE_CLOUD_TENANT_ID>
```

Comment:
We prefer to use "account ID" across our product docs.  Occassionally, we mention that account id is also known as tenant id, but account is first choice!